lua/telescope/config.lua

```diff
@@ -92,6 +92,7 @@ function config.set_defaults(defaults)
   set("layout_strategy", "horizontal")
   set("layout_defaults", {})
 
+  set("externalize", false)
```

Comment:
Can you add documentation for this? Similar how the other config options are documented. e.g. `scroll_strategy`

Also you probably should rebase and pop the docgen PR. (docgen should work again)
Also sorry that i am so late with this review.

Reply:
Thank you for your review.
I have added the documentation.
I have also rebased and successfully generated the documentation.